projects/components/src/time-picker/time-picker.component.ts

```diff
@@ -9,8 +9,8 @@ import { PredefinedTimeService } from '../time-range/predefined-time.service';
   styleUrls: ['./time-picker.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="time-picker">
-      <ht-popover class="time-selector" [closeOnClick]="true">
+    <div class="time-picker" [ngClass]="[this.disabled ? 'disabled' : '']">
```

Comment:
`[ngClass]="{'disabled': this.disabled}"`

Reply:
done